travis/config/protractor/protractor.conf.js

```diff
@@ -0,0 +1,89 @@
+/*jshint jasmine: true, node: true */
```

Comment:
s there a way to refactor this file to share better with vsts/config/protractor/protractor.conf.js? Seems like they're most identical.

Reply:
Yes, open source only. I initially went down the path of trying to figure out shared configs, but it seemed like it could get hairy if specific platforms needed specific things. I can give it another look though.